.github/workflows/run-tests.yml

```diff
@@ -20,8 +20,18 @@ jobs:
       - run: npm install
       - run: npm run check-format
       - run: make web_unit
-      - run: make web_integration
+      - name: make web_integration
+        run: |
+          cd integration_tests_server
+          npm install
+          node index.js &
+          cd ..
+          make web_integration
```

Comment:
Can we move this inside `make web_integration` so that it can also be run locally?